### PR TITLE
startParameterGenerator: fix usage of random state

### PR DESCRIPTION
**Pull request by fkaspar**
Make proper use of numpy random state. By using 'self.generator'
instead of 'np.random' the provided seed get respected.

### CODE REVIEW

File: pyInterface/module/pyPartialWaveFit/_startParameterGenerator.py [hunk 1]
```diff
@@ -130,11 +130,12 @@ def __init__(self, model, seed=None):
...
-		for accMatrix in self.accMatrices:
-			# use accMatrix[0]
-			realAccMatrixTemp = np.zeros(shape=(2*accMatrix[0].shape[0],2*accMatrix[0].shape[0]),dtype=np.float64)
-			realAccMatrixTemp[:-1:2,:] = accMatrix[0].view(np.float64)
-			realAccMatrixTemp[1::2,:] = (1.j*accMatrix[0]).view(np.float64)
+
+		# use first an ONLY dataset: accMatrices[0]
+		for accMatrix in self.accMatrices[0]:
+			realAccMatrixTemp = np.zeros(shape=(2*accMatrix.shape[0],2*accMatrix.shape[0]),dtype=np.float64)
+			realAccMatrixTemp[:-1:2,:] = accMatrix.view(np.float64)
```
Comment (stefan-wallner) [inline pyInterface/module/pyPartialWaveFit/_startParameterGenerator.py]: I guess you want to get the real and imaginary parts here, right?
Why don't you use `np.real` and `np.imag`?

File: pyInterface/module/pyPartialWaveFit/_startParameterGenerator.py [hunk 2]
```diff
@@ -130,11 +130,12 @@ def __init__(self, model, seed=None):
...
+			realAccMatrixTemp[1::2,:] = (1.j*accMatrix).view(np.float64)
```
Comment (stefan-wallner) [inline pyInterface/module/pyPartialWaveFit/_startParameterGenerator.py]: Does this give you -Imag[accMatrix]? 
Is this correct?

Comment (fkaspar) [reply pyInterface/module/pyPartialWaveFit/_startParameterGenerator.py]: I added a comment to the code to explain the expansion.

**Comment by stefan-wallner** [comment]
Thanks a lot!

**Pull request merged**